include/modules/hyprland/workspaces.hpp

```diff
@@ -43,19 +46,27 @@ class Workspaces : public AModule, public EventHandler {
   void update() override;
   void init();
 
+  auto all_outputs() const -> bool { return all_outputs_; }
+  auto show_special() const -> bool { return show_special_; }
```

Comment:
I think it's better to put it in Workspaces. Not individual workspace. Because it's value is same for all workspaces.

Reply:
it is in workspaces

Reply:
Oh, right. Sorry.